sass/_variables.scss

```diff
@@ -1,5 +1,6 @@
 $orange: #ff7900;
 $grey: #6c6c6c;
+$light-grey: #d4d4d4;
 $white: #fff;
 $drk: #000;
 $light-color: #f2f2f4;
```

Comment:
Not related to this PR. I think that it would be better if these would have non-color names(i.e. primary_color, secondary_color, highlight_color, etc).This way if you will need to change the colors of the application you just update the color value for $primary_color from orange to blue. (for example).
Else, like it is now, if you wanna change the application orange color with blue, you will need to update the value of $orange variable, to a blue color hex value, which may not be natural.

Reply:
![image](https://user-images.githubusercontent.com/41734742/61378448-187afd00-a8ae-11e9-8b65-b3b0518bba55.jpg)

Reply:
Thanks for advice. I didn't realize it.